BrainPortal/lib/cbrain_system_checks.rb

```diff
@@ -289,11 +289,10 @@ def self.a050_check_data_provider_cache_wipe
 
   def self.a080_ensure_set_starttime_revision
 
-    # Global for the whole Rails process
-    $CBRAIN_StartTime_Revision = RemoteResource.current_resource.info.revision
-
     #-----------------------------------------------------------------------------
-    puts "C> Current application tag or revision: #{$CBRAIN_StartTime_Revision}"
+    puts  "C> Current application tag or revision: #{CBRAIN::CBRAIN_StartTime_Revision}"
+    print "C> Current Git branch: "
+    CBRAIN::CBRAIN_Git_Branch.present? ? puts(CBRAIN::CBRAIN_Git_Branch) : puts("unknown")
```

Comment:
Again, use .precence?() 

```ruby
print "C> Current Git branch: #{CBRAIN::CBRAIN_Git_Branch.presence || "unknown"}"
```

Why use print followed by puts? This will bypass the custom puts() method used in the boot process.